sprintQuovantis/sprintParam/urls.py

```diff
@@ -4,4 +4,6 @@
 urlpatterns = [
     path('sprint', views.Sprint.as_view(), name='sprint'),
     path('param_sprint', views.ParameterOfSprint.as_view(), name='param_sprint'),
+    path('user_vote', views.UserVote.as_view(), name='user_vote'),
```

Comment:
url paths should have hyphens instead of underscores eg. param-sprint instead of param_sprint